ehitchhiking-ui/src/app/shared/components/active-trip/active-trip.component.html

```diff
@@ -0,0 +1,11 @@
+<div class = "form-row" >
```

Comment:
is it possible to reuse trip component used in the HistoryModalComponent ?

Reply:
fixbug